core/lib/multivm/src/versions/vm_fast/bootloader_state/utils.rs

```diff
@@ -63,13 +63,7 @@ pub(super) fn apply_tx_to_memory(
         (tx_description_offset..tx_description_offset + bootloader_tx.encoded_len())
             .zip(bootloader_tx.encoded.clone()),
     );
-
-    let bootloader_l2_block = if start_new_l2_block {
-        bootloader_l2_block.clone()
-    } else {
-        bootloader_l2_block.interim_version()
```

Comment:
Nit: Can `interim_version()` method be removed after this change? It looks it's not used elsewhere.